src/main/java/mate/academy/service/UserService.java

```diff
@@ -14,7 +14,13 @@ public class UserService {
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
-    public User findByEmail(String email) {
-        return null;
+    public boolean findByEmail(String email, String password) {
```

Comment:
this method should return user, not boolean
please read carefully given comments and make changes to your code